README.md

```diff
@@ -33,11 +33,13 @@ But, this library also gives you the full stack of utilities for scalable RDF st
 - [`jelly-titanium-rdf-api`](https://github.com/Jelly-RDF/jelly-jvm/tree/main/titanium-rdf-api/src) – integration with the minimalistic [Titanium RDF API](https://github.com/filip26/titanium-rdf-api).
   - [![jelly-titanium-rdf-api Scala version support](https://index.scala-lang.org/jelly-rdf/jelly-jvm/jelly-titanium-rdf-api/latest.svg)](https://index.scala-lang.org/jelly-rdf/jelly-jvm/jelly-titanium-rdf-api) [![javadoc](https://javadoc.io/badge2/eu.ostrzyciel.jelly/jelly-titanium-rdf-api_3/javadoc.svg)](https://javadoc.io/doc/eu.ostrzyciel.jelly/jelly-titanium-rdf-api_3)
```

Comment:
Here and in the other packages – the org name name and the package name are wrong – package name now doesn't include _3 at the end.